rules/sinks/third_parties/sdk/google/java.yaml

```diff
@@ -20,14 +20,191 @@ sinks:
       - "(?i)(com[.]google[.]api[.]services[.]analytics|com[.]google[.]android[.]gms[.]analytics).*"
     tags:
 
-  - id: ThirdParties.SDK.Google.Cloud
-    name: Google Cloud
+  - id: ThirdParties.SDK.Google.Cloud.Storage
+    name: Google Cloud Storage
+    domains:
+      - "storage.cloud.google.com"
+    patterns:
+      - "(?i)(com[.]google[.]cloud[.]storage).*"
+    tags:
+
+  - id: ThirdParties.SDK.Google.Cloud.Datastore
+    name: Google Cloud Datastore
     domains:
       - "cloud.google.com"
     patterns:
-      - "(?i)(com[.]google[.]cloud[.]storage|com[.]google[.]cloud[.]dataflow|com[.]google[.]cloud[.]datastore|com[.]google[.]cloud).*"
+      - "(?i)(com[.]google[.]cloud[.]datastore|com[.]google[.]cloud[.]grpc).*"
```

Comment:
@khemrajrathore @HiteshMah-Jan is this rule for gRPC correct?

Reply:
I see this same pattern across rules `com[.]google[.]cloud[.]grpc`, let's use this rule only at a single place

Reply:
modified it